glotaran/analysis/problem.py

```diff
@@ -807,7 +808,7 @@ def _calculate_additional_grouped_penalty(self):
             and self.model.has_additional_penalty_function()
         ):
             self._additional_penalty = self.model.additional_penalty_function(
-                self.parameter, self._full_clp_labels, self.clps, self._full_axis
```

Comment:
This is wrong. the additional penalty function aussumes that all clp are retrieved. Otherways you run in to problems with relations etc.

So the call params should be
`self.parameter, self._clp_labels, self._clps, self._full_axis`

Reply:
I clicked on the wrong side of the review, I mean the changed line.